CHANGELOG.md

```diff
@@ -29,8 +29,10 @@ The types of changes are:
 - Added developer option to disable auto-initialization on FidesJS bundles. [#4900](https://github.com/ethyca/fides/pull/4900)
 - Adding property ID to served notice history and privacy preference history [#4886](https://github.com/ethyca/fides/pull/4886)
 - Adding privacy_center_config and stylesheet fields to the Property model [#4879](https://github.com/ethyca/fides/pull/4879)
+- Include required scopes in API authorization error (403) messages [#4905](https://github.com/ethyca/fides/pull/4905)
 - Adds generic async callback integration support [#4865](https://github.com/ethyca/fides/pull/4865)
 - Ability to `downgrade` the application DB through the `/admin/db` endpoint [#4893](https://github.com/ethyca/fides/pull/4893)
+- Include the scopes required for a given action in `403` response when client does not have sufficient permissions [#1438](https://github.com/ethyca/fidesplus/pull/1438)
```

Comment:
This looks out of place?

Reply:
🤦 trying to move too fast, thanks for catching!

Reply:
ok fixed in https://github.com/ethyca/fides/pull/4905/commits/0f323ab9f6e36b1eaf46bc15c5758070486f99d4 👍